### DISCUSSION TITLE
/blog/2022/05/09/hurdle-lognormal-gaussian-brms/

### DISCUSSION THREAD

**Discussion by andrewheiss** [andrewheiss/ath-quarto#47] [Blog comments]
A guide to modeling outcomes that have lots of zeros with Bayesian hurdle lognormal and hurdle Gaussian regression models | Andrew Heiss 
 https://www.andrewheiss.com/blog/2022/05/09/hurdle-lognormal-gaussian-brms/

**Comment by andrewheiss** [comment]
```header
Originally posted by Mattan S. Ben-Shachar on 2022-05-19 06:09:28
````
There is an important distinction between \`mu\` and \`epred\` when
using mixture models.

See my overly detailed example here (:

[https://github.com/andrewhe\...](https://github.com/andrewheiss/ath-hugo/commit/70197c7389c87b46db6cd61fc1ef8ce7b5c94616#commitcomment-73816081 &quot;https://github.com/andrewheiss/ath-hugo/commit/70197c7389c87b46db6cd61fc1ef8ce7b5c94616#commitcomment-73816081&quot;)

**Comment by andrewheiss** [comment]
```header
Originally posted by Emil Madsen on 2022-10-13 06:28:51
````
This is fantastic Andrew! I had tried using the brms gamma hurdle model
here:
[https://osf.io/preprints/so\...](https://osf.io/preprints/socarxiv/fxqr7/),
but now I think I want to redo the analysis, using all the great ideas
you present here. Thanks for this!

**Comment by andrewheiss** [comment]
```header
Originally posted by Daniel on 2022-10-24 13:40:38
````
Thank you for the great write-up!

**Comment by Erin-Fedewa-NOAA** [comment]
Thank you, thank you Andrew! So well explained and reproducible! I've enjoyed all of your posts and they've been extremely helpful for my own research pursuits and analyses. Kudos for your ability to explain complex processes and analyses so well.

**Comment by bjcresswell** [comment]
What a great post Andrew - thanks heaps! Just a quick question - when you get to the part where you model the hurdle component against life expectancy (https://www.andrewheiss.com/blog/2022/05/09/hurdle-lognormal-gaussian-brms/#hurdle-model-with-additional-terms), you note that we obviously expected this, because the dummy data were set up that way. In the case where you don't know what might be driving the 0s, and where you have multiple predictors, would you say it is a reasonable approach to separately model each predictor against 'hu' and see which one predicts the greatest change in 0s, or is there a way to model hu against all your terms at once? I'm thinking conceptually for the 2 separate models: Response ~ Pred1 + Pred2, hu ~ Pred1 and then Response ~ Pred1 + Pred2, hu ~ Pred2. I can't see a way of doing this in one step? Cheers!
